travis/install-gradle

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh -eux
 
-./gradlew --continue --no-build-cache "$SUBMODULE_PREFIX"assemble
+./gradlew --continue --no-build-cache "$SUBMODULE_PREFIX"assemble verGJF
```

Comment:
Would `verGJF` make more sense in `travis/script-gradle`? That’s where we run other correctness-checking tasks such as `build` (runs all automated tests) and `lintGradle`.

Reply:
After further review of the [Travi CI Job Lifecycle](https://docs.travis-ci.com/user/job-lifecycle/) documentation, I think that `travis/install-gradle` should only handle dependency installation (of which we probably have none), and even `assemble` more properly belongs in `travis/script-gradle`. Let’s leave both `assemble` and `verGJF` in `travis/install-gradle` for now, and plan to move some things over to `travis/script-gradle` later, whenever convenient. This is not a big deal either way.

Reply:
Ok, sounds good, will leave as is for now